Makefile

```diff
@@ -71,8 +71,6 @@ test-sub-p:
 
 .PHONY: hashes
 hashes:
-	python -m aea.cli hash all
-	python -m aea.cli hash all --packages-dir=./tests/data/packages
 	python -m aea.cli packages lock
```

Comment:
Does this now fully subsume above functionality? Needs to be in upgrading guide and release notes.

Reply:
[309db7a](https://github.com/valory-xyz/open-aea/pull/466/commits/309db7ad60ae4c071a0a2f26a33f377b035df95a)